src/style-value.js

```diff
@@ -12,24 +12,23 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(shared, testing) {
+(function(internal, scope, testing) {
 
-  function StyleValue() {
-  }
+  function StyleValue() {}
 
   // TODO: Include parsing logic here.
   StyleValue.parse = function(property, value) {
     if (typeof value == 'string') {
       numberValue = Number.parseFloat(value);
       if (numberValue !== NaN) {
-        return new NumberValue(numberValue);
+        return new scope.NumberValue(numberValue);
```

Comment:
Is this just a stylistic thing? Do we want to be doing this everywhere? I'm wondering if we should even bother having a 'scope' parameter if it's always going to be window.